lib/matplotlib/transforms.py

```diff
@@ -2848,11 +2848,11 @@ def _interval_contains_close(interval, val, rtol=1e-10):
 
     Parameters
     ----------
-    interval : sequence of scalar
-        A 2-length sequence, endpoints that define the interval.
-    val : scalar
+    interval : (float, float)
+        The endpoints of the interval.
+    val : float
         Value to check is within interval.
-    rtol : scalar
+    rtol : float, default: 1e-10
         Tolerance slippage allowed outside of this interval.  Default
         1e-10 * (b - a).
```

Comment:
this sentence is contradictory with the edit two lines above (I know "default" doesn't mean the same in both cases, but still)